src/common/httpClient/testHttpClient.ts

```diff
@@ -21,7 +20,7 @@ export class TestHttpClient implements BasicHttpClient {
     if (resolvedObj) {
       const httpResult: HttpResult<TDto> = {
         body: resolvedObj as TDto,
-        statusCode: resolvedHttpRequest.resolveStatusCode ?? HttpStatus.OK,
+        statusCode: resolvedHttpRequest.resolveStatusCode ?? 200,
```

Comment:
I suggest continue using the HttpStatus package in order to avoid magic numbers

Reply:
The whole idea was to get rid of an 8 year old package that just introduces an unnecessary dependency and increases package bundle size

Reply:
I understand, then let's use 200 here. :)